### PR TITLE
give some guidance on system requirements for hosting Grist

### PR DESCRIPTION
**Pull request by paulfitz**
For simplicity, I stick with hosting Grist as a container. This could be a good place to collect requirements, rules of thumb for scaling, and known issues as we learn them.

### CODE REVIEW

File: help/self-managed.md [hunk 1]
```diff
@@ -400,6 +400,46 @@ based on an export of the SendGrid templates for our SaaS.
...
+runs comfortably served from a Grist container with:
+
+  * 100MB RAM without sandboxing enabled.
+  * 200MB RAM with sandboxing enabled.
+  * 1 CPU.
+
+Memory and CPU requirements will scale with the number of documents
+simultaneously in use by your team.
+
+[Sandboxing](self-managed.md#sandboxing) is an important issue in serving
```
Comment (dsagal) [inline help/self-managed.md:429]: This in-page link isn't working for me (I see at least one other place where it occurs from before this diff and also doesn't work)

**Review by dsagal — COMMENTED** [review]
This is very useful. I found it informative too! Looks good, just something seems off about the anchor links.

File: help/self-managed.md (hunk 1 above)
Comment (paulfitz) [reply help/self-managed.md:429]: Oh good catch, lots of links broke when I rewrote section titles FAQ style... working on fixing.

Comment (paulfitz) [reply help/self-managed.md]: I updated bad anchors in self managed docs. I think I've discovered others, will follow up in a separate PR.

**Review by dsagal — APPROVED** [review]
Thanks! I found two small typos unrelated to this diff:
- in line 227: "The UI elements present is also affected" -> "The UI elements present are also affected"
- in line 271: "you can made" -> "you can make".

**Pull request merged**